homeassistant/components/matter/number.py

```diff
@@ -183,6 +217,24 @@ def _update_from_device(self) -> None:
         ),
         vendor_id=(4874,),
     ),
+    MatterDiscoverySchema(
+        platform=Platform.NUMBER,
+        entity_description=MatterLevelControNumberEntityDescription(
+            key="pump_setpoint",
+            native_unit_of_measurement=PERCENTAGE,
+            translation_key="pump_setpoint",
+            native_max_value=100,
+            native_min_value=0,
+            native_step=10,
```

Comment:
Why do we use step size 10? Shouldn't the step size be 0.5 in accordance with the spec?